src/components/weather-box.js

```diff
@@ -92,15 +92,17 @@ export const WeatherBox = ({ days, locationName }) => {
                                 <Year>
                                     {new Date(day.datetime).getFullYear()}
                                 </Year>
-                                <Temperature>{`${day.temp} ºC`}</Temperature>
+                                <Temperature>
+                                    {day.temp ? `${day.temp} ºC` : 'No data'}
```

Comment:
Para evitar que ponga "No data" si la temperatura es cero #falsy #typeCoercion

```suggestion
                                    {day.temp !== null ? `${day.temp} ºC` : 'No data'}
```